src/main.leo

```diff
@@ -95,6 +105,10 @@ program privx_exchange.aleo {
         };
         Mapping::set(sequence, 0u64, next_id + 1u64);
         Mapping::set(registry, next_id, token_meta);
+        let token_setting: Setting = Setting {
+            claim_amount: 0u64,
```

Comment:
How about adding claim_amount as a parameter in transition register?

Reply:
Maybe add a extra function to change the settings is better